django/core/management/commands/migrate.py

```diff
@@ -313,7 +314,7 @@ def model_installed(model):
                 if self.verbosity >= 1:
                     self.stdout.write("    Running deferred SQL...\n")
                 for statement in deferred_sql:
-                    cursor.execute(statement)
+                    cursor.execute(force_text(statement))
```

Comment:
@claudep yeah this PR has gone a bit stale thanks for heads up. I'll try to rebase and remove Python2 oddities later today.